b_cfn_elasticsearch_cloner/cloner_source/index.py

```diff
@@ -51,52 +73,69 @@ def handler(event: Dict[Any, Any], context: Any) -> None:
     :return: No return.
     """
 
-    logger.info("Starting processing DynamoDB events.")
+    logger.info('Starting processing DynamoDB events.')
+
+    use_embeddings = bool(
+        SAGEMAKER_EMBEDDINGS_KEY and SAGEMAKER_ENDPOINT_NAME and EMBEDDINGS_KEY)
+
+    if use_embeddings:
+        logger.info(f'Performing embeddings.')
+
+    if not use_embeddings:
+        if any([SAGEMAKER_EMBEDDINGS_KEY, SAGEMAKER_ENDPOINT_NAME, EMBEDDINGS_KEY]):
```

Comment:
try to limit code nesting whenever it is possible. Like in this case, you could simply use `and` condition to combine these two lines together.